benchmarks/Lua/storage.lua

```diff
@@ -39,15 +39,15 @@ end
 function storage:build_tree_depth (depth, random)
     self.count = self.count + 1
     if depth == 1 then
-        -- With Lua, an array cannot be pre-allocated.
+        -- With Lua, an array cannot be pre-allocated. (actually yes, you can)
```

Comment:
This doesn't seem to be a great comment any more, could you amend it perhaps to something more suitable?

Reply:
Alright, removing this